msal/tests/Test.MSAL.NET.Integration/Test.MSAL.NET.Integration.csproj

```diff
@@ -19,8 +22,13 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\adal\src\Microsoft.IdentityModel.Clients.ActiveDirectory\Microsoft.IdentityModel.Clients.ActiveDirectory.csproj">
```

Comment:
Please add a comment on why this is needed, i.e. qualified name collisions...

Reply:
to cover unified cache test cases we need to be able create instances of Adal  and Msal in same test

Reply:
Sorry for not making myself clear. When I say "please add a comment", I mean "please add a comment in the code" not "please add a comment in the PR"